play-scala-websocket-example/conf/application.conf

```diff
@@ -13,10 +13,14 @@ pekko {
   }
 }
 
-play.filters.enabled += play.filters.csp.CSPFilter
```

Comment:
This is the reason why the example works now. Because you removed the CSPfilter. If you just remove this line all the other changes are not necessary. The csp directives disallow the javascript files to load. So you removed the csp filter completely -> example works.